noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -513,10 +1010,14 @@
                 ]]>
             </errors>
         </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST3">
         <title>Fonds period containment</title>
+        <description>
+            Tests whether the Fonds' created and finalized dates
```

Comment:
`Tests whether the created and finalized dates of all fonds are within the archival period specified in arkivuttrekk.xml`